server/config/app.js

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import * as bodyParser from 'body-parser';
+
+import logger from 'morgan';
+import userRoutes from './routes/Users';
+import documentRoutes from './routes/Documents';
+import roleRoutes from './routes/Roles';
+
+
+const app = express();
+
+const PORT = process.env.PORT || 4040;
+
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use('/users', userRoutes);
+app.use('/documents', documentRoutes);
+app.use('/roles', roleRoutes);
+
+/* Logout only */
+app.get('/logout', (req, res) => {
+  /* Destroy and blacklist token here */
+  res.status(201).json({
```

Comment:
Thanks @andela-cnwankwo.

The idea is to actually save the token in the database for blacklisting purpose which would mean a status code of `created` right?